src/routers/__tests__/StackRouter-test.js

```diff
@@ -1355,6 +1355,28 @@ describe('StackRouter', () => {
     );
   });
 
+  test('URI encoded string get passed to deep link', () => {
+    const uri = 'people/2018%2F02%2F07';
+    const action = TestStackRouter.getActionForPathAndParams(uri);
+    expect(action).toEqual({
+      routeName: 'person',
```

Comment:
uri has `people`, but `routeName` is `person`, is it correct?

Reply:
yes https://github.com/react-navigation/react-navigation/blob/2187d8fe660a35a579e971cf237634a3bfb60afb/src/routers/__tests__/StackRouter-test.js#L69-L72